### ISSUE TITLE
Username length

### ISSUE THREAD

**Issue by mkrogh** [bogeblad/infoglue#15]
Hi, since the forum is "down" (spammed beyond recognition), I'll "report" the issue here.

I have a problem with the length of usernames in: Management -> SystemUsers -> New User

In the Create new user view there is a validation of the username length (20char) which is rather low. We are currently using email addresses as usernames, and an email can quite easily become 20+ char.

Is there any way to increase the limit in the view? (the db tables ranges usernames from 100char to 255 (to text)), so it is not looking at the DB field lenght.

The create new user view is defined in https://github.com/bogeblad/infoglue/blob/master/conf/cms/actions.xml, but I have not been able to locate any length validation anywhere.

In short can the input validation be increased? (preferably through the interface)
